pkg/resources/aws/eni.go

```diff
@@ -79,6 +79,9 @@ func DescribeENIs(cloud fi.Cloud, clusterName string) (map[string]*ec2.NetworkIn
 		request := &ec2.DescribeNetworkInterfacesInput{
 			Filters: append(filters, statusFilter),
 		}
+		if vpcID != "" {
```

Comment:
nice.

Reply:
actually, here we may as well return. there's no ENI without VPC and it's probably valid for most other things.